apps/teams-test-app/e2e-test-data/pages.json

```diff
@@ -25,6 +25,14 @@
       "expectedAlertValue": "returnFocus called with true",
       "expectedTestAppValue": "Current navigateForward state is true"
     },
+    {
+      "title": "returnFocus With Type API Call - Success",
+      "type": "callResponse",
+      "boxSelector": "#box_returnFocusWithType",
+      "inputValue": 2,
+      "expectedAlertValue": "returnFocus called with false",
+      "expectedTestAppValue": "Current state is 2"
```

Comment:
Is the idea that you will come back and add another e2e test testing that your enum value gets sent to the host after you check in the host-sdk part of the change?

Reply:
changed it expect enum. However since it requires hub sdk test change - I'll mostly keep it behind a hubsdk web version